trajectory_executor/datastructures.py

```diff
@@ -0,0 +1,33 @@
+serial_read_states = {
+  'INIT': 0,
+  'READ_PREAMBLE': 1,
+  'READ_ROLL': 2,
+  'READ_PITCH': 3,
+  'READ_YAW': 4,
+  'READ_L_HIP': 5,
+  'READ_L_KNEE': 6,
+  'READ_R_HIP': 7,
+  'READ_R_KNEE': 8,
+  'READ_L_SHOULDER': 9,
+  'READ_L_ELBOW': 10,
+  'READ_R_SHOULDER': 11,
+  'READ_R_ELBOW': 12,
+  'READ_AT_GOAL': 13,
+  'READ_CHECKSUM': 14,
+}
+
+data_packet_struct = {
+  'roll': 0,
+  'pitch': 0,
+  'yaw': 0,
+  'l_hip': 0,
+  'l_knee': 0,
+  'r_hip': 0,
+  'r_knee': 0,
+  'l_shoulder': 0,
+  'l_elbow': 0,
+  'r_shoulder': 0,
+  'r_elbow': 0,
+  'at_goal': 0,
+  'checksum': 0,
+}
```

Comment:
Will these two dictionaries always be repeated? if so, could they be merged by perhaps making a richer data structure such as a `collections.named_tuple` or a similar data structure?

Reply:
Could you elaborate? I'm not sure I follow. Are you saying to put these two dictionaries together as part of a parent dictionary? i.e.
```
collections = {
    'serial_read_states': {...},
    'data_packet_struct': {...}
}
```

Reply:
You could try something like:

```
_State = namedtuple('State', ['serial', 'data_name', 'order'])
```
and create the dict by something like 
```
states = {
  'roll' = _State('READ_ROLL', 'roll', 2),
}
```
but the more I think about it, that might just be overkill. idk, whatever works!

Reply:
Yea in this case I'll elect to keep it as is for the sake of simplicity. Later down the line if there's time and want, we can try changing it up.